app/models/reviewModels.js

```diff
@@ -63,4 +63,24 @@ const fetchCommentsByReviewId = (review_id) => {
     });
 };
 
-module.exports = { fetchAllReviews, fetchReviewById, fetchCommentsByReviewId };
+const insertCommentByReviewId = (reqBody, review_id) => {
+  const insertQuery = `
+  INSERT INTO comments 
+    (body, review_id, author, votes)
+  VALUES
+    ($1, $2, $3, $4)
+  RETURNING *;
+  `;
+  const insertValues = [reqBody.body, review_id, reqBody.username, 0];
```

Comment:
I like that you're thinking about it but do we need to pass the the votes in as 0? I'm sure the database defaults this value to 0.